### ISSUE TITLE
Sign Distributed Packages

### ISSUE THREAD

**Issue by drGrove** [ripple/rippled#3009]
Currently GitHub releases/tags are signed, but upstream compiled packages (ie. RPM) are not. This allows the mirrors to potentially MITM packages prior to installation.

```
 rpm -qpi ripple-repo-el7.rpm
Name        : ripple-repo
Version     : 1
Release     : 2.el7.centos
Architecture: noarch
Install Date: (not installed)
Group       : Unspecified
Size        : 1367
License     : MIT
Signature   : (none) <-- (This should be not be none)
Source RPM  : ripple-repo-1-2.el7.centos.src.rpm
Build Date  : Mon Apr 11 15:22:25 2016
Build Host  : 2cc2d608e62f
Relocations : (not relocatable)
URL         : http://mirrors.ripple.com/
Summary     : Repository for Ripple Labs RPM packages
Description :
YUM configuration files for installing packages from the Ripple Labs repo

```

Information on signing RPM packages with GPG can be found [here](https://access.redhat.com/blogs/766093/posts/1976693). For most packages, there is a group master key (see [tor](https://2019.www.torproject.org/docs/signing-keys.html.en)).

**Comment by mellery451** [comment]
Thanks for the report. This particular package (ripple-repo) is deprecated and was never signed. That said, our current 1.3 packages are *also* not rpm-signed - this is something we'll be looking into correcting moving forward. I'll keep this issue open until we have a solution. thanks.

**Comment by mellery451** [comment]
packages for 1.5 are now properly signed - will wait to close this until 1.5 is released.

**Comment by carlhua** [comment]
Issue is fixed in 1.5 - closing.
